### ISSUE TITLE
Option to skip sensor completely

### ISSUE THREAD

**Issue by znerd** [SonarSecurityCommunity/dependency-check-sonar-plugin#156]
**Is your feature request related to a problem? Please describe.**
I would like to be able to explicitly skip the Dependency Check sensor during the sonar scan.

**Describe the solution you'd like**
I'd like to be able to explicitly configure certain code basis to indicate the Dependency-Check sensor should _not_ be executed when the SonarQube Scanner is run.

**Describe alternatives you've considered**
Just not providing a file is implicit, it will output a message saying the invocation is “skipped/aborted.”

See also #69.

**Additional context**
Some of the other SonarQube plugins also support a `skip` configuration setting.

Suggested property: `sonar.dependencyCheck.skip` (default value being `false`).

**Comment by Reamer** [comment]
I'll add this feature.

**Comment by gbatalski** [comment]
We have a requirement not to overwrite the dependnecy check result, if current upload to sonarqube doesn't contain the depcheck xml output. I've tried to achieve this via the parameter `sonar.dependencyCheck.skip=true` which works well. But the already existing depcheck report in Sonarqube was also removed. We would like to preserve the last depckeck analysis until the new one created (eg. nightly or weekly), so developer can investigate the problems meanwhile. Is there any way to achieve this?
